_includes/configuration/dockerized-hyperlinking-service.md

```diff
@@ -0,0 +1,134 @@
+
+    ```sh
+    unzip ephox-hyperlinking-docker-starter-kit.zip -d hyperlinking-service-dockerfile
+    ```
+
+6. Copy `ephox-hyperlinking.war` into the directory containing the extracted files, such as:
+
+    ```sh
+    cp ephox-hyperlinking.war hyperlinking-service-dockerfile/
+    ```
+
+4. Navigate into the `hyperlinking-service-dockerfile` folder.
+
+    ```sh
+    cd hyperlinking-service-dockerfile
+    ```
+{% elsif shbundledockerfiles == false %}
+1. Go to [{{ site.accountpage }}]({{ site.accountpageurl }}) > My Downloads
+and download either:
+* _Tiny Enhanced Media Embed_, or
+* _Tiny Link Checker_.
+2. Open a command line and navigate to the directory containing `ephox-hyperlinking_<version>.zip`. Windows Users should open a Bash command line as the Administrator User.
+3. Extract the contents of `ephox-hyperlinking_<version>.zip`, such as:
+
+    ```sh
+    unzip ephox-hyperlinking_<version>.zip -d tinymce-hyperlinking-service
+    ```
+    The structure of the extracted files will be:
+    ```sh
+    tinymce-hyperlinking-service/
+    ├── ephox-hyperlinking-docker-env.conf
+    ├── docker-entrypoint.sh
+    ├── Dockerfile
+    ├── ephox-hyperlinking.war
+    └── generate-jetty-start.sh
+    ```
+4. Navigate into the extracted folder.
+
+    ```sh
+    cd tinymce-hyperlinking-service
+    ```
+{% endif %}
+5. Set the permissions on the extracted files to executable.
+
+    ```sh
+    chmod +x *.sh
+    ```
+6. _Optional_: Edit the `http` configuration settings in `ephox-hyperlinking-docker-env.conf`. The configurable settings are in the `http` section of the file. For example:
+
+    ```
+    http {
+      max-connections = 100
+      max-connections-per-host = 10
+      max-redirects = 10
+      max-retries = 3
+      request-timeout-seconds = 10
+      trust-all-cert = false
+      websphere = {
+        use-ssl-config = true
+      }
+    }
+    ```
+    For information on the `http` configuration setting, see: [Configure server-side components - `http`]({{site.baseurl}}/enterprise/server/configure/#httpoptional).
+1. _Optional_: Update the link-checking cache configuration, as described in [Link Checker self-hosted quick setup]({{site.baseurl}}/enterprise/check-links/#linkcheckerself-hostedquicksetup).
+1. _Optional_: Configure the Enhanced Media Embed Service, as described in [Configure Enhanced Media Embed Server]({{site.baseurl}}/enterprise/embed-media/mediaembed-server-config/).
+1. _Optional_: Configure the service to use a HTTP proxy by updating `ephox-hyperlinking-docker-env.conf`. See:
+[Configure server-side components - proxy]({{site.baseurl}}/enterprise/server/configure/#proxyoptional).
+1. Create an `origins.env` file and specify the Hypertext Transfer Protocol (HTTP) and domain name of sites hosting the TinyMCE editor (`allowed-origins`). Up to 99 origins can be added without editing `ephox-hyperlinking-docker-env.conf`.
+
+    For example:
+
+    ```conf
+    ORIGIN0=[example.net](<http://example.net/>)
```

Comment:
Is this supposed to have the `[]` and `()`, or is it a markdown formatting error?

Reply:
fixed